src/monitor/btcaddrs.go

```diff
@@ -0,0 +1,206 @@
+package monitor
+
+import (
+	"encoding/json"
+	_ "encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/boltdb/bolt"
+	"github.com/sirupsen/logrus"
+	"github.com/skycoin/teller/src/addrs"
+	"github.com/skycoin/teller/src/util/httputil"
+)
+
+type BtcAddressJSON struct {
+	Addresses []string `json:"btc_addresses"`
+}
+
+type CountBtc struct {
+	All  int
```

Comment:
Needs json struct tags e.g. `json:"all"` to enforce lowercase naming for API fields

Reply:
good